File: Homeworks/C#2/Methods/05.LargerThanNeightbours/LargerThanNeighbours.cs
using System;
using System.Linq;

//Problem 5. Larger than neighbours

//Write a method that checks if the element at given position in given array of integers is larger than its two neighbours (when such exist).

class LargerThanNeighbours
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int[] array = new int[n];
        string textNumbers = Console.ReadLine();
        var numbers = textNumbers.Split(' ').ToArray();
        for (int i = 0; i < numbers.Length; i++)
        {
            array[i] = int.Parse(numbers[i]);
        }
        Console.WriteLine(FindLargerElement(array));
    }

    static int FindLargerElement(int[] array)
    {
        int count = 0;
        for (int position = 1; position < array.Length - 1; position++)
        {
            if (array[position] > array[position + 1] && array[position] > array[position - 1])
            {
                count++;
            }
        }
        return count;
    }


}

File: Homeworks/JSFundamentals/Fucntions/largerThanNeighbours.js
var array = [2, 5, 3, 8, 7, 6, 9],
    position = 2;

function checkElementAtPosition(array, position) {
    var i;
    if (position < 1 || position > array.length - 1) {
        return 'This element have not two neighbours';
    } else {
        i = position;
            if (array[i - 1] < array[i] && array[i] > array[i + 1]) {
                return true;
            } else {
                return false;
            }
    }
}

console.log(checkElementAtPosition(array, position));